### PR TITLE
change contributors name

### PR DESCRIPTION
**Pull request by Trial97**

### CODE REVIEW

**Review by timoreo22 — APPROVED** [review]

**Comment by ZekeZDev** [comment]
While we are at this do you reckon you could update mine please. I changed my link lil bit back and currently it links to wrong person.

should be:
```
stream << QString("<p>ZekeZ %1</p>\n").arg(getGitHub("ZekeZDev"));
```

**Comment by Trial97** [comment]
Done but I kept the same order

**Comment by ZekeZDev** [comment]
why am I no longer in the developers section anymore ;-;

File: launcher/ui/dialogs/AboutDialog.cpp [hunk 1]
```diff
@@ -111,7 +111,7 @@ QString getCreditsHtml()
     stream << "<p>TakSuyu &lt;<a href='mailto:taksuyu@gmail.com'>taksuyu@gmail.com</a>&gt;</p>\n";
     stream << "<p>Kilobyte &lt;<a href='mailto:stiepen22@gmx.de'>stiepen22@gmx.de</a>&gt;</p>\n";
     stream << "<p>Rootbear75 &lt;<a href='https://twitter.com/rootbear75'>@rootbear75</a>&gt;</p>\n";
-    stream << "<p>Zeker Zhayard &lt;<a href='https://twitter.com/zeker_zhayard'>@Zeker_Zhayard</a>&gt;</p>\n";
+    stream << QString("<p>ZekeZ %1</p>\n").arg(getGitHub("ZekeZDev"));
```
Comment (TheKodeToad) [inline launcher/ui/dialogs/AboutDialog.cpp]: This person is not ZekeZ

Comment (TheKodeToad) [reply launcher/ui/dialogs/AboutDialog.cpp]: https://github.com/PrismLauncher/PrismLauncher/blob/0700c76c361fa59396cca815103054c7395353eb/launcher/ui/dialogs/AboutDialog.cpp#L80 this person is

Comment (Trial97) [reply launcher/ui/dialogs/AboutDialog.cpp]: ups

**Review by TheKodeToad — APPROVED** [review]

**Review by TayouVR — APPROVED** [review]

**Review by txtsd — APPROVED** [review]

**Pull request merged**